enki/PhysicalEngine.cpp

```diff
@@ -1145,16 +1145,25 @@ namespace Enki
 		{
 			// init physics interactions
 			for (ObjectsIterator i = objects.begin(); i != objects.end(); ++i)
-				(*i)->initPhysicsInteractions(overSampledDt);
+			{
+				if(SkipPhysicsObjectsList.find(*i) == SkipPhysicsObjectsList.end())
+					(*i)->initPhysicsInteractions(overSampledDt);
+			}
 			
 			// collide objects together
 			unsigned iCounter, jCounter;
 			iCounter = 0;
 			for (ObjectsIterator i = objects.begin(); i != objects.end(); ++i)
 			{
+				if(SkipPhysicsObjectsList.find(*i) != SkipPhysicsObjectsList.end())
+					continue;
+
 				jCounter = 0;
 				for (ObjectsIterator j = objects.begin(); j != objects.end(); ++j)
 				{
+					if(SkipPhysicsObjectsList.find(*j) != SkipPhysicsObjectsList.end())
```

Comment:
Enki's coding convention is to put a space between `if` and the opening parenthesis: `if (`.